activitysim/abm/models/vehicle_type_choice.py

```diff
@@ -0,0 +1,444 @@
+# ActivitySim
+# See full license in LICENSE.txt.
+
+import logging
+
+import pandas as pd
+import numpy as np
+import itertools
+import os
+
+from activitysim.core.interaction_simulate import interaction_simulate
+from activitysim.core import simulate
+from activitysim.core import tracing
+from activitysim.core import config
+from activitysim.core import inject
+from activitysim.core import pipeline
+from activitysim.core import expressions
+from activitysim.core import logit
+from activitysim.core import assign
+from activitysim.core import los
+
+from activitysim.core.util import assign_in_place
+from .util import estimation
+
+logger = logging.getLogger(__name__)
+
+
+def get_combinatorial_vehicle_alternatives(alts_cats_dict, model_settings):
```

Comment:
Suggest adding a unit test for this function.

Reply:
done.  See the abm/test/test_misc/test_vehicle_type_alternatives.py script.